src/argo.h

```diff
@@ -14,9 +14,14 @@
 
 /**
  * @brief initialize ArgoDSM system
- * @param size the desired size of the global memory in bytes
+ * @param argo_size The desired size of the global memory in bytes, or 0. If the
+ *                  value is specified as 0, then the value in environment
+ *                  variable @ref ARGO_MEMORY_SIZE is used instead.
+ * @param cache_size The desired size of the local cache in bytes, or 0. If the
+ *                   value is specified as 0, then the value in environment
+ *                   variable @ref ARGO_CACHE_SIZE is used instead.
```

Comment:
If `ARGO_CACHE_SIZE` is uninitialized, then a fallback value is used. Should this be documented here?

Reply:
Fixed in 7d2d3c7